osu.Game.Rulesets.Mania/Objects/Drawables/DrawableHoldNoteBody.cs

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
+// See the LICENCE file in the repository root for full licence text.
+
+#nullable disable
```

Comment:
Consider this temporary for now. `DrawableHitObject` is not properly annotated for its nullable ctor argument, and I don't want to use `null!` in this case.